apps/web/src/app/docs/showcase/page.tsx

```
@@ -0,0 +1,122 @@
+        expires. Jira, Cursor execution, visitor connections, and arbitrary
+        website automation are outside this showcase.
+      </p>
+      <h2 className="mt-12 text-2xl font-semibold">
+        Existing QuickChat integrations
+      </h2>
+      <p className="mt-4 leading-7">
+        Existing props and legacy text chat remain supported. Opt into the new
+        scoped controller by rendering <code>Conversation</code> directly or
+        supplying the optional <code>conversation</code> prop to{" "}
+        <code>QuickChat</code>. This release does not silently migrate an
+        existing application's transcript or endpoint.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)